packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.5.0+21
+
+* Changes fallback resolution strategies for camera use cases to look for a higher resolution if first the desired
+  resolution is unavailable and then also a lower resolution is unavailable.
```

Comment:
Completely optional reword, but I personally found the original phrasing confusing

```suggestion
* Changes fallback resolution strategies for camera use cases to look for a higher resolution if neither the desired
  resolution nor any lower resolutions are available.
```